lms/djangoapps/mobile_api/v1/users/serializers.py

```diff
@@ -0,0 +1,26 @@
+"""
+Serializer for user v1 API
+"""
+
+from rest_framework import serializers
+from mobile_api.users.serializers import CourseEnrollmentSerializer as CourseEnrollmentSerializerBase
+from openedx.features.course_duration_limits.access import get_user_course_expiration_date
+from student.models import CourseEnrollment
+
+
+class CourseEnrollmentSerializer(CourseEnrollmentSerializerBase):
+    """
+    Extend base CourseEnrollmentSerializer, add expiration field
+    """
+    expiration = serializers.SerializerMethodField()
+
+    def get_expiration(self, model):
+        """
+        Returns expiration date for a course, if any or null
+        """
+        return get_user_course_expiration_date(model.user, model.course)
+
+    class Meta(object):
+        model = CourseEnrollment
+        fields = ('expiration', 'created', 'mode', 'is_active', 'course', 'certificate')
```

Comment:
could you paste the values of 
http://localhost:18000/api/mobile/v0.5/users/audit/course_enrollments/
and 
http://localhost:18000/api/mobile/v1/users/audit/course_enrollments/
in the PR just so we can compare?

Reply:
added to the PR description

Reply:
thanks!  
for v0.5 is that what it looks like without your changes?  
Based on your changes I would assume it would not return the course because it is expired?

also, we probably want to make the v0.5 changes behind the feature flag from my PR?  
Otherwise, I think most courses on learners' mobile dashboards will disappear before we turn content gating on.

Reply:
yes that's correct, the v0.5 output I showed was for a course that is not expired. if expired the output would have been `[]`

I'll add the feature flag check as well, good catch!

Reply:
@bill-filler That course is expired for me locally and you can see the start date is well into the past, so I think it should be expired?

Reply:
yes it is. I just manually changed the code to return ACCESS_GRANTED so I could show the full output in the comments as that is what I thought you wanted to see.